compiler/rustc_ast_lowering/src/lib.rs

```diff
@@ -2826,6 +2828,16 @@ impl<'a, 'hir> LoweringContext<'a, 'hir> {
             )
         }
     }
+
+    fn item_attrs(&self, def_id: DefId) -> Vec<ast::Attribute> {
+        if let Some(_local_def_id) = def_id.as_local() {
+            // FIXME: This doesn't actually work, items doesn't include everything?
+            //self.items[&hir::ItemId { def_id: local_def_id }].attrs.into()
+            Vec::new()
```

Comment:
Yea I think this is a non-issue. Any function that calls a rustc_legacy_const_generics function within the same crate can just use const generics as users will never see that attribute.